src/Microsoft.Data.SqlClient/netfx/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -629,9 +632,6 @@
     <Compile Include="Microsoft\Data\SqlClient\Server\sqlser.cs" />
     <Compile Include="Microsoft\Data\Sql\SqlGenericUtil.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(BuildSimulator)' == 'true'">
-    <Compile Include="Microsoft\Data\SqlClient\SimulatorEnclaveProvider.cs" />
-  </ItemGroup>
```

Comment:
is removing the condition intentional?

Reply:
Yes, in phase 2, the newly named  `NoneAttestationEnclaveProvider` will now actually be used within the driver when `Attestation Protocol = None;` is specified. Before it was only used for testing purposes when the simulator flag was set.